test/verify/check-shell-multi-machine-key

```diff
@@ -139,7 +139,7 @@ class TestMultiMachineKeyAuth(testlib.MachineCase):
         try:
             m1.execute("ps xa | grep ssh-agent | grep -v grep")
         except subprocess.CalledProcessError:
-            assert False, "No running ssh-agent found"
+            assert False, "No running ssh-agent found"  # noqa: PT015 assertion always fails
```

Comment:
That should work with `self.fail("msg")`, no? See https://docs.python.org/3/library/unittest.html#unittest.TestCase.fail